sparql/sparql11/protocol/manifest.ttl

```diff
@@ -1,961 +1,1430 @@
+                ht:elementName "application/sparql-query"
+              ])
+          ]);
+        ht:httpVersion "1.1";
+        ht:methodName "POST";
+        ht:resp [
+          a ht:Response;
+          ht:body [
+            a cnt:ContentAsText;
+            cnt:characterEncoding "UTF-8";
+            cnt:chars true
+          ];
+          ht:headers ([
+              a ht:RequestHeader;
+              ht:fieldName "Content-Type";
+              ht:fieldValue "application/sparql-results+xml, application/sparql-results+json";
+              ht:headerElements ([
+                  a ht:HeaderElement;
+                  ht:elementName "application/sparql-results+xml"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "application/sparql-results+json"
+                ])
+            ]);
+          ht:statusCodeValue "2XX",
+            "3XX"
+        ]
+      ])
+  ];
+  mf:name "query via POST directly" .
+
+<#query_post_form> a mf:ProtocolTest;
+  dawg:approval dawg:Approved;
+  dawg:approvedBy <http://www.w3.org/2009/sparql/meeting/2012-11-20#resolution_3>;
+  mf:action [
+    a ht:Connection;
+    ht:connectionAuthority "www.example";
+    ht:requests ([
+        a ht:Request;
+        ht:absolutePath "/sparql?query=ASK%20%7B%7D&default-graph-uri=http%3A%2F%2Fkasei.us%2F2009%2F09%2Fsparql%2Fdata%2Fdata0.rdf";
+        ht:httpVersion "1.1";
+        ht:methodName "POST";
+        ht:resp [
+          a ht:Response;
+          ht:body [
+            a cnt:ContentAsText;
+            cnt:characterEncoding "UTF-8";
+            cnt:chars true
+          ];
+          ht:headers ([
+              a ht:RequestHeader;
+              ht:fieldName "Content-Type";
+              ht:fieldValue "application/sparql-results+xml, application/sparql-results+json";
+              ht:headerElements ([
+                  a ht:HeaderElement;
+                  ht:elementName "application/sparql-results+xml"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "application/sparql-results+json"
+                ])
+            ]);
+          ht:statusCodeValue "2XX",
+            "3XX"
+        ]
+      ])
+  ];
+  mf:name "query via URL-encoded POST" .
+
+<#update_base_uri> a mf:ProtocolTest;
+  dawg:approval dawg:Approved;
+  dawg:approvedBy <http://www.w3.org/2009/sparql/meeting/2012-11-20#resolution_3>;
+  mf:action [
+    a ht:Connection;
+    ht:connectionAuthority "www.example";
+    ht:requests ([
+        a ht:Request;
+        ht:absolutePath "/sparql";
+        ht:body [
+          a cnt:ContentAsText;
+          cnt:characterEncoding "UTF-8";
+          cnt:chars "update=CLEAR%20SILENT%20GRAPH%20%3Chttp%3A%2F%2Fexample.org%2Fprotocol-base-test%2F%3E%20%3B%20INSERT%20DATA%20%7B%20GRAPH%20%3Chttp%3A%2F%2Fexample.org%2Fprotocol-base-test%2F%3E%20%7B%20%3Chttp%3A%2F%2Fexample.org%2Fs%3E%20%3Chttp%3A%2F%2Fexample.org%2Fp%3E%20%3Ctest%3E%20%7D%20%7D"
```

Comment:
Original test encoded update using direct-POST request body, not form-url-encoding.